CRM/Report/Form/Case/Detail.php

```diff
@@ -56,9 +64,12 @@ public function __construct() {
             'no_display' => TRUE,
             'required' => TRUE,
           ],
+          'case_id' => [
```

Comment:
@mattwire this doesn't have 'type' - it might give e-notices? It's an additional field on the report I think

Reply:
@eileenmcnaughton Thanks - added